nodeJS/express/introduction_to_express.md

```diff
@@ -0,0 +1,98 @@
+### Introduction
+
+In the previous lesson, you got up and running with Node. You learned how to read and write data from files on the server, and use those to serve up a multi-page website. You may have found this a somewhat verbose and cumbersome process and may have even wondered how on Earth we would manage more complex procedures.
+
+In this course, we are going to be using a backend framework called [Express](https://expressjs.com/). It will handle some of the more cumbersome things for us, and make the things we did previously far simpler. Express itself is an intentionally barebones and unopinionated framework; it allows us to do many things how we want, and to extend it with only the features we need. While this gives us great flexibility in how we do things, it can be a little tricky deciding between multiple viable options at times.
+
+Going forward, we will be diving into how we can use Express in various ways, such as to create a full stack application utilising the `Model View Controller` (MVC) pattern, as well as to create a REST API just like ones you will have used before for things such as the Weather App project or the React Shopping Cart. Along the way, we will be learning about things like databases, authentication, and much more. There is a lot to take in, so take it steady and do not be afraid to ask our community for help in [our Discord server](https://discord.gg/theodinproject)!
+
+### Lesson overview
+
+This section contains a general overview of topics that you will learn in this lesson.
+
+- Describe what Express is.
+- Describe the relationship between Express and Node.
+- Set up a basic Express server.
+- Describe how an incoming request is processed and ultimately responded to by the server.
+- Describe what a middleware function is.
+- Describe how to use `nodemon` to automatically restart your application when a change is made.
+
+### Setting up Express
+
+Let's make a basic Express app. Inside a new directory, start by running `npm init` and filling in each option to create a `package.json`. Once that's created, we can install the Express dependency.
```

Comment:
We could just ask learners to run `npm init -y`. Your call.

Reply:
We could. I also just noticed that `package.json` defaults to `"main": "index.js"`.
I don't think this should be any issue really since AFAIK it's only for creating an npm package so it knows what entry point to use when the package is imported.

I guess fine to leave `main` at that and just don't mention it?

`npm init -y` would probably be easier.